diff -ur pyserial-2.7.1.orig/serial/tools/miniterm.py pyserial-2.7.1/serial/tools/miniterm.py
--- a/serial/tools/miniterm.py	2017-01-17 15:41:57.368085174 +0100
+++ b/serial/tools/miniterm.py	2017-01-17 15:35:22.794864438 +0100
@@ -8,6 +8,7 @@
 # SPDX-License-Identifier:    BSD-3-Clause
 
 import codecs
+import datetime
 import os
 import sys
 import threading
@@ -46,6 +47,7 @@
             self.byte_output = sys.stdout.buffer
         else:
             self.byte_output = sys.stdout
+        self.timestamp = False
         self.output = sys.stdout
 
     def setup(self):
@@ -65,6 +67,9 @@
 
     def write(self, text):
         """Write string"""
+        if self.timestamp:
+            now = datetime.datetime.now()
+            text = text.replace('\n', now.strftime('\n[%Y-%M-%d %X] '))
         self.output.write(text)
         self.output.flush()
 
@@ -578,6 +590,9 @@
             self.dump_port_settings()
         #~ elif c == '\x01':                       # CTRL+A -> cycle escape mode
         #~ elif c == '\x0c':                       # CTRL+L -> cycle linefeed mode
+        elif c in 'tT':                         # T -> display timestamp at beginning of line
+            with self.console:
+                self.console.timestamp = not self.console.timestamp
         elif c in 'pP':                         # P -> change port
             with self.console:
                 try:
@@ -675,6 +690,7 @@
 --- Toggles:
 ---    {rts:7} RTS   {dtr:7} DTR   {brk:7} BREAK
 ---    {echo:7} echo  {eol:7} EOL
+---    {timestamp:7} timestamp
 ---
 --- Port settings ({menu} followed by the following):
 ---    p          change port
@@ -695,7 +711,8 @@
            upload=key_description('\x15'),
            repr=key_description('\x01'),
            filter=key_description('\x06'),
-           eol=key_description('\x0c'))
+           eol=key_description('\x0c'),
+           timestamp=key_description('\x74'))
 
 
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
